src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferencesCascadeDirection.cs

```diff
@@ -1,33 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-
-namespace Microsoft.CodeAnalysis.FindSymbols
-{
-    [Flags]
-    internal enum FindReferencesCascadeDirection
```

Comment:
no need for thsi enum anymore.  the old approach had hte engine building out the inheritance graph by calling into finders to continually expand in a specific direction (or in both directions).  this is now handled directly in the new symbol set code.